site/content/integrate/apps/devenv/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Dev environment"
+heading: "Admin's Guide to Deploying Apps"
+description: "TODO"
+weight: 100
+---
+
+TODO
```

Comment:
What will this file be used for?

Reply:
To describe the docker-based dev environment, I think it's missing in the docs

Reply:
https://github.com/mattermost/mattermost-plugin-apps/tree/master/dev contains some docs on that. Should we copy them over?